src/index.ts

```diff
@@ -1,26 +1,25 @@
 import { ReplaySubject } from 'rxjs/ReplaySubject'
 
-const subjects = new WeakMap();
+const subjects = new WeakMap()
```

Comment:
I should really have typed this `WeakMap`.

Reply:
I have added a typing for this. I'm not sure that you'll like a 'SubjectByProp' type alias name though